src/util-error.h

```diff
@@ -34,6 +34,7 @@ typedef enum {
     SC_ERR_PCRE_GET_SUBSTRING,
     SC_ERR_PCRE_COMPILE,
     SC_ERR_PCRE_STUDY,
+    SC_ERR_PCRE_JITSTACK,
```

Comment:
convention is to only append new codes so codes stay static

Reply:
Will move to the end of the enum list.

Reply:
> convention is to only append new codes so codes stay static

I'm curious why.  Is it possible to get the code, instead of the name output in the logging where static may actually matter?